### PR TITLE
OSSM-4016 Changes to better support maistra-assist

### PR DESCRIPTION
**Pull request by dgn**

### CODE REVIEW

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/maistra-dev/istio-ubi9-operator
-  newTag: "3.0"
+  newName: registry.gitlab.com/dgrimm/istio/istio-ubi9-operator
```
Comment (jwendell) [inline config/manager/kustomization.yaml]: Is this right?

File: config/default/kustomization.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-namespace: maistraoperator-system
+namespace: istio-operator
 
 # Value of this field is prepended to the
 # names of all resources, e.g. a deployment named
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: maistraoperator-
+# namePrefix: istiooperator-
```
Comment (jwendell) [inline config/default/kustomization.yaml:9]: Intended change?

File: config/manager/kustomization.yaml (hunk 1 above)
Comment (dgn) [reply config/manager/kustomization.yaml]: nope. thanks :)

File: config/default/kustomization.yaml (hunk 2 above)
Comment (dgn) [reply config/default/kustomization.yaml:9]: yeah, I needed to get rid of the prefix

Comment (jwendell) [reply config/default/kustomization.yaml:9]: ok, let me put in a different way: Should we remove this field entirely (including the comments above it)? Or just commenting it is enough meaning this is a temporary change?

**Review by jwendell — APPROVED** [review]
minor nit, feel free to ignore

File: config/default/kustomization.yaml (hunk 2 above)
Comment (dgn) [reply config/default/kustomization.yaml:9]: yeah it's a good question. I decided to leave it in while we're figuring things out. We'll need a big cleanup anyway, there's a lot of commented out stuff, and a lot of boilerplate we don't need. I'll create a ticket to cover that

**Pull request merged**